docs/team/ngquyduc.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: Nguyen Quy Duc's Project Portfolio Page
+---
+
+### Project: HobbyList
+
+HobbyList is a desktop application used for tracking various hobby activities. The user interacts with it using a CLI, and it has a GUI created with JavaFX.
+
+Given below are my contributions to the project.
+
+* **New Feature**: to be added soon
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=ngquyduc&breakdown=true)
+
+* **Project management**:
+  * to be added soon
+
+* **Enhancements to existing features**:
+  * to be added soon
+
+* **Documentation**:
+  * User Guide:
+    * to be added soon
+  * Developer Guide:
+    * to be added soon
+
+* **Community**:
+  * to be added soon
+
+* **Tools**:
+  * to be added soon
```

Comment:
I think you need a new line here to pass the tests.

Reply:
I add a new line and it pass the tests.